packages/smooth_app/lib/functions/launchURL.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:intl/intl.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:basic_utils/basic_utils.dart';
+
+class Launcher{
+
+  ///
+  /// ifOFF (is Open food facts)
+  /// when true add language identifier at start
+  ///
+  /// !!!
+  /// NO  https://de.openfoodfacts.org/...
+  /// YES https://   openfoodfacts.org/...
+  /// !!!
+  ///
+
+
+
+  void launchURL(String url, bool isOFF ) async{
+
+    String openURL;
+
+
+    if(isOFF){
+      final String locale = Intl.getCurrentLocale();
+      openURL = StringUtils.addCharAtPosition(url, '${locale ?? 'en'}.', 8);
```

Comment:
in the url xx.openfoodfacts.org, xx is the country code, not a language code.  en.openfoodfacts.org does not exist in fact.

Reply:
Thanks for pointing out that. Actually, I did not plan to include this commit in the pull request. 
I will have a look on it. 
I think I will switch to `Localizations.localeOf(context).countryCode`.